Makefile

```diff
@@ -185,16 +198,23 @@ ASSET_FILES_OUT := $(foreach f,$(ASSET_FILES_XML:.xml=.c),$f) \
 
 UNDECOMPILED_DATA_DIRS := $(shell find data -type d)
 
+BASEROM_SEGMENTS_DIR := baseroms/gc-eu-mq-dbg/segments
```

Comment:
now that I tried the thing I'm thinking this is probably intended, forgot to read again my comments

Reply:
Yeah, currently the build system doesn't know how to extract segments from anything but the debug ROM. This will change soon, but I added a comment to hopefully prevent confusion